### PR TITLE
[Retribtuion Paladin]Basic Ability Modules + Updates

### PR DESCRIPTION
**Pull request by LordSkeletor**
Updated Wake of Ashes to new event handlers and improved inefficient casts
Updated Judgment module to new event handlers and added inefficient casts
NEW: Consecration Talent Module- Highlight inefficient casts
NEW: Hammer of Wrath Talent Module- Highlight inefficient casts
NEW: Crusader Strike Module- Highlight inefficient casts
NEW: Blade of Justice Module- Highlight inefficient casts
New modules are very basic for now.

### CODE REVIEW

File: src/parser/paladin/retribution/modules/talents/Consecration.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import SPELLS from 'common/SPELLS';
+
+class Consecration extends Analyzer {
+  static dependencies = {
```
Comment (Sharrq) [inline src/parser/paladin/retribution/modules/talents/Consecration.js]: empty dependencies

File: src/parser/paladin/retribution/modules/talents/Consecration.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  };
+
+  wasteHP = false;
```
Comment (Sharrq) [inline src/parser/paladin/retribution/modules/talents/Consecration.js]: `const variable = value`

File: src/parser/paladin/retribution/modules/talents/HammerofWrath.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import SPELLS from 'common/SPELLS';
+
+class HammerofWrath extends Analyzer {
+  static dependencies = {
```
Comment (Sharrq) [inline src/parser/paladin/retribution/modules/talents/HammerofWrath.js]: empty dependencies

File: src/parser/paladin/retribution/modules/talents/HammerofWrath.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  };
+
+  wasteHP = false;
```
Comment (Sharrq) [inline src/parser/paladin/retribution/modules/talents/HammerofWrath.js]: `const variable = value`

File: src/parser/paladin/retribution/modules/talents/WakeofAshes.js [hunk 5]
```diff
@@ -1,96 +1,118 @@
...
-  };
+    static dependencies = {
+        abilityTracker: AbilityTracker,
+        spellUsable: SpellUsable,
+    };
 
-  totalHits = 0;
-  badCasts = 0;
-  wakeCast = false;
+    totalHits = 0;
```
Comment (Sharrq) [inline src/parser/paladin/retribution/modules/talents/WakeofAshes.js]: `const variable = value`

**Comment by LordSkeletor** [comment]
Updated with requested changes. Thank you.

**Pull request merged**